pontoon/administration/files.py

```diff
@@ -129,21 +131,18 @@ def modify_entity_mine(self, id, value):
 """ End monkeypatching """
 
 
-def get_locale_paths(project, locale):
+def get_locale_paths(project, locale, source_paths, source_directory):
```

Comment:
So instead of walking the FS you're just doing a string replacement and then checking for existence. Why the change? I wouldn't expect `os.walk` to be a source of much slowness.